code/modules/mob/interactive.dm

```diff
@@ -1216,27 +1216,25 @@
 
 	if(shouldTryHeal == 1)
 		for(var/mob/living/carbon/human/C in nearby)
-			if(istype(C,/mob/living/carbon/human)) //I haven't the foggiest clue why this is turning up non-carbons but sure here whatever
```

Comment:
I sure hope this doesn't bite me in the ass somehow, even though it literally shouldn't.